backend/src/test/java/edu/pl/masi/yellow/model/LoginTokenTest.java

```diff
@@ -0,0 +1,37 @@
+package edu.pl.masi.yellow.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import static junit.framework.TestCase.*;
+
+public class LoginTokenTest {
+    @Test
+    public void InvalidTokenTest() {
```

Comment:
Please be consistent in test naming. 
This test and in line 17 ends with 'Test'. Maybe every test should ends with 'Test' or without it?